server/src/pages/homepage/model.js

```diff
@@ -1,7 +1,8 @@
 exports.createModel = function(req) {
   let model = {
     title: "Firefox Screenshots",
-    showMyShots: !!req.deviceId
+    showMyShots: !!req.deviceId,
+    staticUrl: req.config.staticUrl
```

Comment:
Also when you are setting it there, you can normalize the URL.  Specifically `staticUrl` shouldn't end with a `/`, and if it's set that way it should be removed (like `staticUrl.replace(/\/*$/, "")`).